src/test/java/teammates/test/cases/ui/browsertests/InstructorCourseEnrollPageUiTest.java

```diff
@@ -90,7 +90,8 @@ private void testEnrollAction() throws Exception {
                 // A student with no comment
                 + "Section 3 | Team 3 |Frank Galoe | frank.g.tmms@gmail.tmt |\n"
                 // A new student with name containing accented characters
-                + "Section 1 | Team 1|José Gómez | jose.gomez.tmns@gmail.tmt | This student name contains accented characters\n";
+                + "Section 1 | Team 1|José Gómez | jose.gomez.tmns@gmail.tmt"
+                        + " | This student name contains accented characters\n";
```

Comment:
Ordinarily I'd ask that the whitespace be added to the string in the earlier line, but in this case leaving the whitespace on this line instead looks better :open_mouth: